docs/topics/gradle.md

```diff
@@ -966,6 +966,55 @@ Here is a complete list of options for Gradle tasks:
 | `target` | Generate JS files for specific ECMA version | "v5" | "v5" |
 | `typedArrays` | Translate primitive arrays to JS typed arrays |  | true |
 
+#### Example of additional arguments usage
+
+Use the attribute `freeCompilerArgs` to supply additional (including experimental) compiler arguments. You can add a single 
+argument to this attribute or a list of arguments divided by a comma:
+
+<tabs group="build-script">
+<tab title="Kotlin" group-key="kotlin">
+
+```kotlin
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
```

Comment:
```suggestion
import org.jetbrains.kotlin.gradle.dsl.KotlinCompile
```

Reply:
Are other such fixes correct? Check, please. And what is about other classes from the `tasks` package? Where they moved to `dsl` too?

Reply:
> Check, please.

Done

> And what is about other classes from the tasks package? Where they moved to dsl too?

Not yet. Also `dsl.KotlinCompile` is deprecated in Kotlin 1.8.0 release :sweat_smile: